### DISCUSSION TITLE
Lazy loading components blocking other Livewire round-trips

### DISCUSSION THREAD

**Discussion by br13an** [livewire/livewire#7954] [1. Help]
I've seem to run into a problem, which I'm not sure of is a bug. Livewire seems to be blocking server requests until the lazy loading component is done loading. I cant interact with the other components on the page until it loads. Isolating or Lazyloading doesn't seem to make a difference. 

Wrapping all my components with a root div inside aswell as outside of the <livewire:component> doesn't work.
Adding lazy to all of the components in the tree doesn't work. I did it with defered loading with wire:init on Livewire 2, but that has been deprecated. 

As an example I've got the following 'main page'.
```html
@can('read') 
    <div class="text-lg text-gray-600 font-bold uppercase mt-8 md:mt-0">Specs</div>
    <div>
        <livewire:v-main-card :vehicle="$v" wire:key="v-card-{{$v->id}}"/>
    </div>
@endcan

<!-- This component wont work until the first one has completed the API request -->
@can('read workshop')
    <div class="text-lg text-gray-600 font-bold uppercase">T</div>
    <div>
        <livewire:technical-card :v="$v->v_id"/>
    </div>
@endcan
```
My main card looks like:

```html
<div>
   Some code ...
   @if($v['manually_created'] == false)
    <div>
      <livewire:image-component lazy :v="$v" wire:key="img-over-{{$v['id']}}-{{time()}}"/>
    </div>
  @endif
</div>
```
This image component does some API requests (which are slow) to retrieve some images from an external source. Which I've got in my ImageComponent.php;

```php
class ImageComponent extends Component
{

    public function mount()
    {
        // This is the API request handled in a different file.
        $this->images = $this->getPictures($this->v['v_id']);
    }

    public function placeholder()
    {
        return <<<'HTML'
            <div role="status">
                <svg aria-hidden="true" class="w-6 h-6 mr-2 text-gray-200 animate-spin dark:text-gray-600 fill-blue-600" viewBox="0 0 100 101" fill="none" xmlns="http://www.w3.org/2000/svg">
                    <path/>
                </svg>
                <span class="sr-only">Loading...</span>
            </div>
            HTML;
    }
}
```

**Comment by Geovanek** [comment]
I have the same problem, I have a component that I added lazy by default
```php
#[Lazy(isolate: false)]
final class Day extends Component
```

And neither the @can or @if properties or any other to use in the blades do not work.

If I remove `lazy` it works normally.
